[2024-12-06T21:24:12.785+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-06T21:24:12.799+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: homework_zone_dag.local_to_gcs manual__2024-12-06T21:24:01.598190+00:00 [queued]>
[2024-12-06T21:24:12.810+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: homework_zone_dag.local_to_gcs manual__2024-12-06T21:24:01.598190+00:00 [queued]>
[2024-12-06T21:24:12.811+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2024-12-06T21:24:12.822+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): local_to_gcs> on 2024-12-06 21:24:01.598190+00:00
[2024-12-06T21:24:12.853+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=5176) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-12-06T21:24:12.858+0000] {standard_task_runner.py:72} INFO - Started process 5177 to run task
[2024-12-06T21:24:12.970+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'homework_zone_dag', 'local_to_gcs', 'manual__2024-12-06T21:24:01.598190+00:00', '--job-id', '141', '--raw', '--subdir', 'DAGS_FOLDER/homework_zone_dag.py', '--cfg-path', '/tmp/tmp7o66fdl7']
[2024-12-06T21:24:13.051+0000] {standard_task_runner.py:105} INFO - Job 141: Subtask local_to_gcs
[2024-12-06T21:24:13.970+0000] {task_command.py:467} INFO - Running <TaskInstance: homework_zone_dag.local_to_gcs manual__2024-12-06T21:24:01.598190+00:00 [running]> on host 39ecaeffba2c
[2024-12-06T21:24:14.324+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='homework_zone_dag' AIRFLOW_CTX_TASK_ID='local_to_gcs' AIRFLOW_CTX_EXECUTION_DATE='2024-12-06T21:24:01.598190+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-06T21:24:01.598190+00:00'
[2024-12-06T21:24:14.327+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-06T21:24:14.980+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-12-06T21:24:15.027+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-06T21:24:15.030+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=homework_zone_dag, task_id=local_to_gcs, run_id=manual__2024-12-06T21:24:01.598190+00:00, execution_date=20241206T212401, start_date=20241206T212412, end_date=20241206T212415
[2024-12-06T21:24:15.169+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-06T21:24:15.258+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-12-06T21:24:15.287+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
